Worktrolly/worktrolly
# feat(dev/activity): added an activity collection in database.

### Functionality:
Adding Activity in collection

### Solution:
Based on the proposed schema, an activity collection is added to the database, to track each activity of the task.

### Risk level:

### How to test:
1. ng serve
2. create a task and perform operations
3. observe the updated collection on firebase console

An "Activity" collection is created: 
![Screenshot (14)](https://user-images.githubusercontent.com/52309262/105864617-215a9c00-6018-11eb-8e19-acea64f499d7.png)

An "Action" collection is created under each taskId:
![Screenshot (15)](https://user-images.githubusercontent.com/52309262/105864673-30d9e500-6018-11eb-8332-7b6e29024187.png)
